taf/updater/updater.py

```diff
@@ -345,9 +353,14 @@ def _update_target_repositories(
             if is_git_repository:
                 repository.fetch(branch=branch)
             if old_head is not None:
-                new_commits_on_repo_branch = repository.all_fetched_commits(
-                    branch=branch
-                )
+                if not only_validate:
```

Comment:
could we reverse and just say `if only_validate`? using not and else makes my head hurt